packages/fcl/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 ## Unreleased
 
+- 2021-11-30 -- Allow apps to add opt-in wallets in Discovery with config.
+
+```javascript
+import { config } from "@onflow/fcl"
+
+config({
+  "discovery.authn.endpoint": "https://fcl-discovery.onflow.org/api/testnet/authn",
+  "discovery.authn.optIn": ["0x123"]
```

Comment:
I'm ok with optIn, but might suggest something like `supported` or something.
So the idea, is that if a service is of type `optIn`, app must specify to include in it's config, yeah?

Reply:
Yeah if a service is marked as `optIn` then you must explicitly say in the config you want it to show.

Reply:
Yea i'm thinking `include` and `exclude` might be better options